app/scripts/modules/core/react.module.ts

```diff
@@ -0,0 +1,46 @@
+import {module} from 'angular';
+import 'ngimport';
+import * as ReactGA from 'react-ga';
+import * as angular from 'angular';
+import IInjectorService = angular.auto.IInjectorService;
+
+import {STATE_EVENTS} from 'core/state.events';
+import {SETTINGS} from 'core/config/settings';
+
+// Initialize React Google Analytics
+if (SETTINGS.analytics.ga) {
+  ReactGA.initialize(SETTINGS.analytics.ga, {});
+}
+
+export const REACT_MODULE = 'spinnaker.react';
+module(REACT_MODULE, [
+  'bcherny/ngimport',
+  STATE_EVENTS,
+]).run(function ($injector: any) {
+  // Make angular services importable and Convert angular components to react
+  ReactInjector.flush($injector);
+});
+
```

Comment:
no longer maintain an explicit master registry of "things that want the injector"